Cargo.toml

```diff
@@ -1,10 +1,9 @@
 [package]
 name = "async_mutex"
-version = "0.1.0"
-authors = ["Do Duy <juchiast@gmail.com>"]
+version = "0.1.1"
+authors = ["real <real@freedomlayer.org>"]
```

Comment:
You should add yourself and Kamyuen Tse (kevin.xjy@gmail.com) as authors. After all you two made this work.